changelog/unreleased/bugfix-access-details-shown-for-all-space-members

```diff
@@ -0,0 +1,4 @@
+Bugfix: Access details shown for all space members
```

Comment:
Wait, this PR doesn't need its own changelog item. Could you please remove it and add the PR link to the other changelog item where you introduced the share details drop?

Reply:
sure